libbeat/docs/outputconfig.asciidoc

```diff
@@ -79,7 +79,7 @@ output.elasticsearch:
 
 If the Elasticsearch nodes are defined by `IP:PORT`, then add `protocol: https` to the yaml file.
 
-[source,yaml]
+["source","yaml",subs="attributes,callouts"]
```

Comment:
Good catch! You can probably just use `subs="attributes"` since this example doesn't have callouts. Doesn't hurt either way, tho....

Reply:
Oops. You're right, not sure why I added callouts in there 🐠. I guess I'll leave it since it doesn't hurt (but mostly so I don't have to wait for 5 successful tests again 😄)